src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -472,8 +470,11 @@ def _check_pid_from_log_file(self, pid, abort_on_warning, leak_wf, show_memleaks
                 if line.is_calloc():
                     pointer = line.calloc_pointer()
                     if pointer in regions:
+                        # Report both the old and new allocation points here.
                         show_line(regions[pointer], 'NORMAL',
-                                  'new allocation seen for same pointer')
+                                  'new allocation seen for same pointer (old)')
+                        show_line(line, 'NORMAL',
+                                  'new allocation seen for same pointer (new)')
```

Comment:
This is the only functional change, right? What does this actually do?

Reply:
In the NLT errors in Jenkins it reports both the old allocation point "regions[pointer]" as well as the new one "line" making it easier to trace the error in the logs.